internal/pkg/envconf/env.go

```diff
@@ -0,0 +1,17 @@
+package envconf
+
+import (
+	"github.com/PhamDuyKhang/littledetective/internal/pkg/flog"
+	"github.com/kelseyhightower/envconfig"
+)
+
+func Load(envstruct interface{}) {
```

Comment:
I don't see anywhere you call this func.
Also, I quite doubt that this function will run normally, as you have to pass a reference to the `envconfig.Process()`, not interface.